.github/workflows/auto-merge.yml

```
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     if: github.actor == 'dependabot[bot]'
     steps:
-      - uses: ahmadnassri/action-dependabot-auto-merge@v2.4
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2.6
```

Unpinning to major version like their README example
```suggestion
      - uses: ahmadnassri/action-dependabot-auto-merge@v2
```

---

Agreed.